packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -816,6 +817,7 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
     Curve curve = Curves.ease,
     ScrollPositionAlignmentPolicy alignmentPolicy = ScrollPositionAlignmentPolicy.explicit,
     RenderObject? targetRenderObject,
+    ScrollableState? previousScrollable,
```

Comment:
This does not look like something we want to do. What does this mean in the context of a single scroll position, no nested scroll view?